src/container.js

```diff
@@ -1,45 +1,44 @@
 const debug = require('debug')('restinga:container');
```

Comment:
This whole file diff is wrong. Editing the container isn't intended. We need to remove it.